pyswervedrive/module.py

```diff
@@ -221,6 +221,7 @@ def set_velocity(self, vx: float, vy: float, *, absolute_rotation: bool = False)
         else:
             # figure out the most efficient way to get the module to the desired direction
             # current_unwound_azimuth = self.measured_azimuth
+            self.read_steer_pos()  # TODO this is for debugging the unwinding issue, remove once fixed
```

Comment:
Don't you just want to uncomment the above commented line? If this was the source of our problems, then clearly we don't want to do this at all.